Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserTableViewProxyModel.cpp

```diff
@@ -104,12 +114,22 @@ namespace AzToolsFramework
             case Qt::DisplayRole:
                 if (orientation == Qt::Horizontal)
                 {
-                    section += section ? aznumeric_cast<int>(AssetBrowserEntry::Column::Type) - 1 : 0;
-                    return tr(AssetBrowserEntry::m_columnNames[section]);
+                    int section2 = section ? section + aznumeric_cast<int>(AssetBrowserEntry::Column::Type) - 1 : 0;
+                    if (GetShowSearchResultsMode())
+                    {
+                        if (section < 3)
```

Comment:
could you please change these to intuitively named constants for these instead of having magic 1, 2, 3 numbers?

Reply:
Changed to reflect column names.